testing/tests/DevExpress.ui.widgets.editors/checkbox.tests.js

```diff
@@ -160,6 +161,74 @@ QUnit.module('Checkbox', function() {
             assert.equal($element.find(ICON_SELECTOR).height(), initHeight, 'icon height is not resized');
         });
 
+        if(isRenovation) {
+            QUnit.test('checkbox icon must resize according to the "iconWidth" and "iconHeight" options', function(assert) {
+                const $element = $('#checkBox').dxCheckBox();
+                const instance = $element.dxCheckBox('instance');
+                const newSize = 50;
+
+                instance.option('iconWidth', newSize);
+
+                assert.strictEqual($element.find(ICON_SELECTOR).outerWidth(), newSize, 'icon width is resized');
```

Comment:
we can get the element once before the asserts

Reply:
done